### PR TITLE
feat: alert template message pt3

### PR DESCRIPTION
**Pull request by wrn14897**
Use Handlebars to build up alert template and trigger notifyChannel method (with `@` prefix)

----

| <a href="https://ellipsis.dev" target="_blank">Ellipsis</a> | :rocket: This PR description was created by [Ellipsis](https://www.ellipsis.dev) for commit d3e4c00612d4089850293646a0c0a18b35d1546b.  | 
|--------|--------|

### Summary:
This PR enhances the alert template building process, supports slack_webhook in the notifyChannel method, introduces new functions, removes the `extractChannels` function, updates the `checkAlerts.test.ts` file, all within the `checkAlerts.ts` file, and modifies the `webhook.ts` model to include a WebhookService enum and updates the schema.

**Key points**:
- Enhances alert template building using Handlebars in `checkAlerts.ts`
- Supports slack_webhook in notifyChannel method in `checkAlerts.ts`
- Introduces new functions for template rendering and alert message body construction in `checkAlerts.ts`
- Removes the `extractChannels` function in `checkAlerts.ts`
- Updates the test file `checkAlerts.test.ts`
- Modifies the `webhook.ts` model to include a WebhookService enum and updates the schema

----
Generated with :heart: by [ellipsis.dev](https://www.ellipsis.dev)

### CODE REVIEW

File: packages/api/src/tasks/checkAlerts.ts [hunk 1]
```diff
@@ -177,40 +217,70 @@ export const buildAlertMessageTemplateTitle = ({
...
             value,
           )
-            ? 'exceeds'
-            : 'falls below'
+            ? alert.threshold_type === 'above'
+              ? 'exceeds'
+              : 'falls below'
+            : alert.threshold_type === 'above'
+            ? 'falls below'
+            : 'exceeds'
```
Comment (wrn14897) [inline packages/api/src/tasks/checkAlerts.ts:247]: found this existing bug 😅

File: packages/api/src/tasks/checkAlerts.ts [hunk 2]
```diff
@@ -133,6 +131,48 @@ type AlertMessageTemplateDefaultView = {
...
+    hdxLink: string;
+    title: string;
+    body: string;
+  };
+  teamId: string;
+}) => {
+  switch (channel) {
+    case 'slack_webhook': {
+      const webhook = await Webhook.findOne({
+        _id: id,
```
[the remark this answers is not in the document]
Comment (wrn14897) [reply packages/api/src/tasks/checkAlerts.ts:152]: hmm...it seems AI is not there yet

[the remark this answers is not in the document]
Comment (hbrooks) [reply packages/api/src/tasks/checkAlerts.ts:152]: Cofounder of Ellipsis here, thanks for this feedback!

[the remark this answers is not in the document]
Comment (wrn14897) [reply packages/api/src/tasks/checkAlerts.ts:152]: hey @hbrooks , what a surprise! Thanks for checking it out

**Review by MikeShi42 — DISMISSED** [review]

**Pull request merged**